packages/data-context/src/actions/WizardActions.ts

```diff
@@ -392,79 +403,8 @@ export class WizardActions {
   private ensureDir (type: 'component' | 'e2e' | 'fixtures') {
     return this.ctx.fs.ensureDir(path.join(this.projectRoot, 'cypress', type))
   }
-
```

Comment:
Moved to `packages/scaffold-config`, trying to consolidate all the scaffolding code in one place.